src/routes/profile.route.test.ts

```diff
@@ -41,4 +41,43 @@ describe('profile', () => {
       expect(response.body).toHaveProperty('uuid');
     });
   });
+
+  describe('Update profile route', () => {
+    let accessToken: string;
+
+    beforeAll(async () => {
+      const testUser = {
+        email: 'test@gmail.com',
+        password: '123',
+      };
+
+      const response = await supertest(server)
+        .post('/api/auth/login')
+        .send(testUser)
+        .expect(200);
```

Comment:
we need to create a user before login, then use this API. we will use db containers to run tests (not yet implemented due to the typeorm issue).